### DISCUSSION TITLE
Mobile view: Titles are cut off

### DISCUSSION THREAD

**Discussion by yo-less** [0x2E/fusion#31] [Q&A] [answered]
I love the design of fusion (have just started self-hosting with rss and this project is by far the most beautiful and modern rss aggregator), but I was wondering if you're considering redesigning the mobile view. In my instance, every title is cut off, I can neither fully read the title of any article nor fully identify the feed that any article is from. Maybe changing it to two lines would solve that problem? If custom CSS was allowed I could try to change the font size, but as it stands, it's not very user-friendly (albeit beautiful to look at).

Also, I would prefer to be able to sort the articles by publication date. Maybe this isn't such an issue once every feed is automatically refreshed, but as it stands right now, I can only see the articles of the last feed I added.

And finally, any chance this might be extended to support multiple users?

![Screenshot_2024-11-12-18-45-50](https://github.com/user-attachments/assets/ed8875ba-711b-4266-b3d5-5f2652a9f42e)

**Comment by 0x2E** [comment, marked as answer]
Hey, glad to see you like it.

1. UI will be refactored in next version. But I have no bandwidth recently :( 
2. Posts of newly added feed are displayed at the top of the list. This could be refactored along with new UI as well.
3. No plan for multiple users, at least in v1.

**Reply by yo-less** [reply]
Thank you for the quick reply, I'd love to see an UI update and a different post order, I keep looking at other self-hosted RSS solutions but nothing is as modern and clean as Fusion. At the same time, it's hard to use Fusion on a phone at the moment and it would be great if all my posts would be sorted by publication date as well.
Oh, and if possible, I'd love to change the tab name as I'm hosting two different instances now to allow for multiple users (so no support for that feature doesn't really disturb me all that much). Keep up the great work and thanks again.
